server/rest/src/test/java/org/infinispan/rest/AuthenticationTest.java

```diff
@@ -52,7 +53,8 @@ public void afterMethod() throws Exception {
    public void shouldAuthenticateWhenProvidingProperCredentials() throws Exception {
       //given
       SecurityDomain securityDomainMock = mock(SecurityDomain.class, new ThrowsExceptionClass(SecurityException.class));
-      doReturn(mock(Principal.class)).when(securityDomainMock).authenticate(eq("test"), eq("test"));
+      SimpleUserPrincipal userPrincipal = new SimpleUserPrincipal("test");
```

Comment:
This change seems unrelated?

Reply:
Actually no. In order to be able to log the user I had to actually return the Principa from the Authenticatorl. This broke the mock !